docs/resources/Channel.md

```diff
@@ -86,6 +87,14 @@ Represents a guild or DM channel within Discord.
 | LATEST_ACTIVITY | 0     | Sort forum posts by activity                                   |
 | CREATION_DATE   | 1     | Sort forum posts by creation time (from most recent to oldest) |
 
+###### Forum Layout Types
+
+| Flag            | Value | Description                                                    |
+| --------------- | ----- | -------------------------------------------------------------- |
+| NOT_SET         | 0     | No default has been set for forum channel                      |
```

Comment:
if it has no default value, how does it display for users by default? list view still?

Reply:
Yes, on the UI it shows as "List" when it's NOT_SET. Because "List" is the current default value.

Reply:
Not necessary, but it could be helpful to include that posts will display as list in this scenario as well